docs/.vitepress/theme/index.ts

```diff
@@ -1,11 +1,18 @@
 import DefaultTheme from 'vitepress/theme-without-fonts'
 import './custom.css'
-import { Theme } from 'vitepress';
+import { Theme, useData, useRoute } from 'vitepress';
 import VersionSwitcher from "../components/VersionSwitcher.vue";
+import codeblocksFold from 'vitepress-plugin-codeblocks-fold'; // import method
+import 'vitepress-plugin-codeblocks-fold/style/index.css'; // import style
```

Comment:
what is this plugin used for?
Does that influence other codeblocks, and if so, do we want that?

Reply:
Update:
It influences *all* codeblocks, see new comment.